Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+
+--
+-- TOC entry 2784 (class 2604 OID 18215)
+-- Name: Small Molecule Section SML_ID; Type: DEFAULT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Section" ALTER COLUMN "SML_ID" SET DEFAULT nextval('"MzTab-M"."Small Molecule Section_SML_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2789 (class 2604 OID 18194)
+-- Name: FeatureXML ID; Type: DEFAULT; Schema: XML; Owner: postgres
+--
+
+ALTER TABLE ONLY "XML"."FeatureXML" ALTER COLUMN "ID" SET DEFAULT nextval('"XML"."FeatureXML_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2788 (class 2604 OID 18205)
+-- Name: TraML ID; Type: DEFAULT; Schema: XML; Owner: postgres
+--
+
+ALTER TABLE ONLY "XML"."TraML" ALTER COLUMN "ID" SET DEFAULT nextval('"XML"."TraML_ID_seq"'::regclass);
+
+
+--
+-- TOC entry 2799 (class 2606 OID 17262)
+-- Name: Metadata_Proteomics Metadata_Proteomics_pkey; Type: CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Metadata_Proteomics"
+    ADD CONSTRAINT "Metadata_Proteomics_pkey" PRIMARY KEY ("mzTab-ID");
+
+
+--
+-- TOC entry 2805 (class 2606 OID 17310)
+-- Name: PSM Section PSM Section_pkey; Type: CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."PSM Section"
+    ADD CONSTRAINT "PSM Section_pkey" PRIMARY KEY ("PSM_ID");
+
+
+--
+-- TOC entry 2803 (class 2606 OID 17308)
+-- Name: Peptide Section Peptide Section_pkey; Type: CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Peptide Section"
+    ADD CONSTRAINT "Peptide Section_pkey" PRIMARY KEY (accession);
+
+
+--
+-- TOC entry 2801 (class 2606 OID 17280)
+-- Name: Protein section Protein section_pkey; Type: CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Protein section"
+    ADD CONSTRAINT "Protein section_pkey" PRIMARY KEY (accession);
+
+
+--
+-- TOC entry 2793 (class 2606 OID 17717)
+-- Name: Metadata_metabolomics Metadata_metabolomics_pkey; Type: CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Metadata_metabolomics"
+    ADD CONSTRAINT "Metadata_metabolomics_pkey" PRIMARY KEY ("mzTab-ID");
+
+
+--
+-- TOC entry 2823 (class 2606 OID 18234)
+-- Name: Small Molecule Evidence (SME) Section Small Molecule Evidence (SME) Section_pkey; Type: CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Evidence (SME) Section"
+    ADD CONSTRAINT "Small Molecule Evidence (SME) Section_pkey" PRIMARY KEY ("SME_ID");
+
+
+--
+-- TOC entry 2825 (class 2606 OID 18254)
+-- Name: Small Molecule Feature (SMF) Section Small Molecule Feature (SMF) Section_SME_ID_REFS_key; Type: CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Feature (SMF) Section"
+    ADD CONSTRAINT "Small Molecule Feature (SMF) Section_SME_ID_REFS_key" UNIQUE ("SME_ID_REFS");
```

Comment:
Are the REFS here actually unique?  I am thinking that it would be possible to have multiple features map to different SME_IDs?

Reply:
i set them to unique